gradle.properties

```diff
@@ -1,7 +1,7 @@
 android.enableD8=true
 
-VERSION_NAME=2.8.0
-VERSION_CODE=27
+VERSION_NAME=2.8.1
```

Comment:
can you please explain the reasons for changing this?

Reply:
Just for update version